rpm/resources/application.properties

```diff
@@ -7,25 +7,25 @@ queueTopicPattern=^testConsumerTopic-*$
 # Number of consumers created to the consumer groups
```

Comment:
please split the properties of ingress library (kafka) to different file from egress library(hdfs) and leave strictly application configuration here

Reply:
Split the properties to different files in commit 25c2def099a5b3fdef6686e86858874cacbdd305